msg/HelperHeading.msg

```diff
@@ -1,4 +1,4 @@
 # Direction the bow of the boat is pointing towards
 # Unit: degrees, 0° north, increases CW
-# Range: 0 <= heading_degrees < 360
+# Range: -180 < heading_degrees <= 180
```

Comment:
should be `heading`, not `heading_degrees`

Reply:
Nice catch! This is fixed now :)